### ISSUE TITLE
[FR] Red light for fiber testing

### ISSUE THREAD

**Issue by RichiH** [carrotIndustries/hubble#1]
It would be nice if you could optionally add a red light source to test fibers with in the next revision. Ideally with configurable blink patterns in case you test more than one.

**Comment by 9er** [comment]
We thought about it, but the LED (which is stupidly bright) won. Problem with a light source is that you need a connector, and of course the light source and lens and everything.

Maybe you could mod an old SFP with a red laser? Or is something like that already on the market maybe?

**Comment by RichiH** [comment]
If we can get a certain volume going, I am pretty certain we can make a drop shipment of pre-assembled things from China work at little cost; problem would be coordination between everyone interested in those.

**Comment by 9er** [comment]
That would be pretty cool. Volume will most certainly be an issue. Also the laser source: I've never seen a TOSA-style red light laser, and also, the required 5-10 dBm might be an issue.

Not sure if this is worth it, since you can just buy a cheap fiber fault finder for ~10 bucks. One more thing to carry around tho.

**Comment by RichiH** [comment]
Agreed, this is not about cost, it's about one-tool-for-ALL-the-things.

**Comment by carrotIndustries** [comment]
To me as well, the fiber fault finder in an SFP form factor seems like the more sensible approach as this keeps the main unit itself nice and small.
